media/js/base/core-datalayer-init.js

```diff
@@ -11,8 +11,6 @@
     var analytics = Mozilla.Analytics;
     var client = Mozilla.Client;
     var dataLayer = (window.dataLayer = window.dataLayer || []);
-    var firefoxDetailsComplete = false;
```

Comment:
I didn't get an okay to stop collecting all firefoxDetails just the FxA ones.  DS wants to keep collecting release channel (if you know of another way to determine release channel let me know).

Probably easiest just to narrow the scope of this PR to the FxADetails but if you want to pair down the FxDetails to just release channel I'm happy to review that too ;)

Reply:
This PR doesn't stop collecting firefoxDetails, we just don't need this variable any longer :)

Reply:
Oh I see it now!